src/wp-content/themes/twentyseventeen/inc/block-patterns.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * Twenty Twenty Theme: Block Patterns
+ *
+ * @package Twenty Seventeen
+ * @since 2.4
+ */
+
+/**
+ * Register Block Pattern Category.
+ */
+if ( function_exists( 'register_block_pattern_category' ) ) {
+
+	register_block_pattern_category(
+		'twentyseventeen',
+		array( 'label' => __( 'Twenty Seventeen', 'twentyseventeen' ) )
+	);
+}
+
+/**
+ * Register Block Patterns.
+ */
+if ( function_exists( 'register_block_pattern' ) ) {
+	register_block_pattern(
+		'twentyseventeen/large-heading-with-button',
+		array(
+			'title'      => __( 'Large Heading with Button', 'twentyseventeen' ),
+			'categories' => array( 'twentyseventeen' ),
+			'content'    => '<!-- wp:heading {"level":1,"textColor":"black","style":{"typography":{"fontSize":50}}} -->
+            <h1 class="has-black-color has-text-color" style="font-size:50px">' . __( 'The content of your <a href="#">Static Front Page</a> is displayed here. This is a great place to add your call to action with a brief message.' ) . '</h1>
+            <!-- /wp:heading -->
+
+            <!-- wp:buttons -->
+            <div class="wp-block-buttons"><!-- wp:button {"borderRadius":0,"className":"is-style-fill"} -->
+            <div class="wp-block-button is-style-fill"><a class="wp-block-button__link no-border-radius">' . __( 'Our Services' ) . '</a></div>
+            <!-- /wp:button --></div>
+            <!-- /wp:buttons -->',
+		)
+	);
+
+	register_block_pattern(
+		'twentyseventeen/images-with-text-and-link',
+		array(
+			'title'      => __( 'Images with Text & Link', 'twentyseventeen' ),
+			'categories' => array( 'twentyseventeen' ),
+			'content'    => '<!-- wp:spacer -->
+            <div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+            <!-- /wp:spacer -->
+            <!-- wp:columns -->
+            <div class="wp-block-columns"><!-- wp:column -->
+            <div class="wp-block-column">
+			<!-- wp:image {"className":"size-large"} -->
+			<figure class="wp-block-image size-large"><img src="' . get_template_directory_uri() . '/assets/images/stripes.jpg" alt="Black Stripes"/></figure>
+			<!-- /wp:image -->
+            <!-- wp:heading {"textColor":"black","style":{"typography":{"fontSize":45}}} -->
+            <h2 class="has-black-color has-text-color" style="font-size:45px">Black Stripes</h2>
+            <!-- /wp:heading -->
+            <!-- wp:paragraph {"textColor":"black","style":{"typography":{"lineHeight":"1.8"}}} -->
+            <p class="has-black-color has-text-color" style="line-height:1.8">' . __( 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.' ) . '</p>
```

Comment:
I wonder what the best way to handle lorem ipsum is… it's not something that would be translated since it's kind of gibberish. Maybe a translator comment that this is placeholder content and should be replaced with the lorem-ipsum equivalent?